src/spd-notification-worker/src/test/java/ca/bc/gov/open/pssg/rsbc/spd/notification/worker/OutputNotificationConsumerTest.java

```diff
@@ -151,7 +151,7 @@ public void setUp() throws JAXBException {
         Mockito
                 .doReturn(DpsDocumentResponse.errorResponse(STATUS_CODE_ERROR))
                 .when(documentServiceMock)
-                .storeDocument(ArgumentMatchers.argThat(x -> x.getFileName().equals(getFileInfo(CASE_1API_ERROR_CODE).getImageReleaseFileName())));
+                .storeDocument(ArgumentMatchers.argThat(x -> x.getFileName().equals(CASE_1API_ERROR_CODE)));
```

Comment:
what is CASE_1

Reply:
It's just a trigger to create different scenarios with the mock objects